xpti/CMakeLists.txt

```diff
@@ -63,5 +74,15 @@ set(CMAKE_BINARY_DIR ${CMAKE_SOURCE_DIR}/lib/${CMAKE_BUILD_TYPE})
 set(EXECUTABLE_OUTPUT_PATH ${CMAKE_BINARY_DIR})
 set(LIBRARY_OUTPUT_PATH ${CMAKE_BINARY_DIR})
 
-include_directories(${CMAKE_SOURCE_DIR}/include)
+if (LLVM_BINARY_DIR)
+  file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/include/xpti
```

Comment:
Does this create the xpti parent directory in the destination or copy its contents?

Reply:
Yes, it copies the entire directory, so that the final path will be something like `/install/prefix/include/xpti/xpti_trace_framework.h`

Reply:
Right, this is just for unification for me to be able to set up tests from llvm-test-suite the same way for developer configuration and installed compiler. First is what most developers use locally, and the latter is what's run in our CI.

Reply:
Got it. Thanks.

This looks OK, but there are some minor caveats with this sort of directory copying. For example, if, after the initial configuration, the developer updates the headers in the sources this may not be reflected in the build directory after a build. This might be confusing behavior, depending on expectations.

Does that sound reasonable?

Reply:
@tcreech-intel that is actually a very good point! Thanks! I have re-implemented this part through custom target.

Reply:
Thanks -- the new arrangement looks good.